--- src/version.c.timestamp	2012-08-04 11:42:18.917118806 +0300
+++ src/version.c	2012-08-04 11:42:29.117118814 +0300
@@ -48,7 +48,7 @@
     strcat(longVersion, ")");
 }
 # else
-char	*longVersion = VIM_VERSION_LONG_DATE __DATE__ " " __TIME__ ")";
+char	*longVersion = VIM_VERSION_LONG_DATE "and built in OBS, see rpm -q --info packagename for more information" ")";
 # endif
 #else
 char	*longVersion = VIM_VERSION_LONG;
