diff --git a/src/os_unix.h b/src/os_unix.h
index 53b31f813..59f9dedfc 100644
--- a/src/os_unix.h
+++ b/src/os_unix.h
@@ -208,10 +208,10 @@ typedef struct dsc$descriptor   DESC;
  * Unix system-dependent file names
  */
 #ifndef SYS_VIMRC_FILE
-# define SYS_VIMRC_FILE "$VIM/vimrc"
+# define SYS_VIMRC_FILE "/etc/vimrc"
 #endif
 #ifndef SYS_GVIMRC_FILE
-# define SYS_GVIMRC_FILE "$VIM/gvimrc"
+# define SYS_GVIMRC_FILE "/etc/gvimrc"
 #endif
 #ifndef DFLT_HELPFILE
 # define DFLT_HELPFILE	"$VIMRUNTIME/doc/help.txt"
